packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -872,6 +875,7 @@ class IrisGrid extends Component<IrisGridProps, IrisGridState> {
     const { model } = this.props;
     this.initState();
     this.startListening(model);
+    this.startListeningForPartitionChanges(model);
```

Comment:
I'd probably break down the current `startListening`/`stopListening` methods, e.g.

```
startListening(model: IrisGridModel): void {
  this.startListeningForEvents(model);
  this.startListeningForPartitionChanges(model);
}
```
As you're calling both startListening methods everywhere, may as well combine it into just one top level one.